guides/v2.3/graphql/reference/categories.md

```diff
@@ -78,6 +78,10 @@ Attribute | Data type | Description
 
 This `CategoryTree` object contains information about the next level of subcategories of the category specified in the original query.
 
+Attribute | Data type | Description
+--- | --- | ---
+`children` | CategoryTree | Child categories tree
```

Comment:
I think I omitted this table on purpose, because `categoryTree` can be  nested many times until Magento determines the query is too complex. I'll accept this, but I need you to make some changes,

*. `CategoryTree` is an array, so wrap it with [] symbols
* Rewrite the description to something like "An array containing the next level of subcategories."